travels_dashboard.Rmd

```diff
@@ -417,6 +420,23 @@ output$travelerInput <- renderUI({
   }
 })
 
+# Create the actual downloadButton for travel edges graph data
+output$download_edges_graph <- renderUI( {
+  downloadButton("downBtnEdgesGraph", "Download .csv", style = "width:100%;")
+})
+
+# Add download handling
+datadownlad_reactive <- reactiveFileReader(intervalMillis = 1000, session = NULL, filePath = 
```

Comment:
An error is appearing here because the file travel_edges_graph.csv is not being created. Indeed, it should be created. It is necessary to have an approach on which the function travel_edges() is called with different parameters depending on the chosen dataset.

Reply:
Now it is solved. The generality of travel_edges() needs to be considered in a later stage